lib/codit/population/networks/regions.py

```diff
@@ -56,7 +56,7 @@ class LSOA(Place):
     """
     with smart_open.open(POPULATION_LSOA_CSV) as fh:
         LSOAs = pd.read_csv(fh)
```

Comment:
General comment, this is a bit confusing as LSOA is a variable. PEP states:
* variables should be lowercase (snake_case)
* class types should be start with an upper case letter (CamelCase)
* constants should be UPPERCASE.